src/CMakeLists.txt

```diff
@@ -196,9 +203,14 @@ if (CURSES)
     endif ()
 
     target_include_directories(cataclysm-common PUBLIC
-        ${CURSES_INCLUDE_DIR} ${NCURSES_INCLUDE_DIRS} ${ZLIB_INCLUDE_DIRS}
+        ${CURSES_INCLUDE_DIR} ${NCURSES_INCLUDE_DIRS}
```

Comment:
is this correct? I don't think we want to drop the includes (and I don't think `target_link_libraries` can work as a substitute for this)?

Reply:
`target_link_libraries` indeed is a substitute for this. Its name is a "trap" becasue [imported targets](https://cmake.org/cmake/help/v3.16/manual/cmake-buildsystem.7.html#imported-targets) like ZLIB::ZLIB define both include path and binary library files location in their properties.